static/app/utils/profiling/platforms.tsx

```diff
@@ -34,6 +34,10 @@ export function getDocsPlatformSDKForPlatform(
     return 'node';
   }
 
+  if (platform === 'javascript-nextjs') {
+    return 'node';
+  }
```

Comment:
Do we want to show some kind of disclaimer that this will only profile the API side of things?

Reply:
I'll add nextjs install docs to the docs and change the platform, just want to not show "unsupported" first